### PR TITLE
Initial hyp3 paper edits

### PR DESCRIPTION
**Pull request by jtherrmann**
These are my initial edits to the HyP3 paper.

One edit corrects important factual information: HyP3 was initially created in 2016 (not 2017) by undergraduate students (none of whom, including myself, were affiliated with UAF).

The remaining edits are just grammar/wording improvements and could be ignored.

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -41,18 +41,18 @@ TO DO
 # HyP3 Description
 
 ## Development History
-HyP3 grew out of a need for a standardized pipeline to perform image processing workflows at the Alaska Satellite Facility (ASF) Distributed Active Archive Center (DAAC). HyP3 was initially created in 2017 by a group of undergraduate University of Alaska Fairbanks students interning at ASF. These students created a prototype version of HyP3 that was designed to handle workflows that ran on-premise and in the AWS cloud. This prototype was iterated upon for several more years, and by 2019 it was capable of performing many SAR-specific image processing workflows.
+HyP3 grew out of a need for a standardized pipeline to perform image processing workflows at the Alaska Satellite Facility (ASF) Distributed Active Archive Center (DAAC). HyP3 was initially created in 2016 by a group of undergraduate students interning at ASF. These students created a prototype version of HyP3 that was designed to handle workflows running on-premises and in the AWS cloud. This prototype was iterated upon for several more years, and by 2019 it was capable of performing many SAR-specific image processing workflows.
```
Comment (hjkristenson) [inline paper/paper.md:44]: https://www.govloop.com/on-premise-vs-on-premises-the-debate-and-resolution/
Hot topic (I had no idea, and it's very fascinating!). If we go with the approach outlined in this article, we should probably un-hyphenate, too: 
"handle workflows running on premises".

File: paper/paper.md [hunk 2]
```diff
@@ -41,18 +41,18 @@ TO DO
...
 
-By 2019 it was clear that re-designing HyP3 to take advantage of evolving technologies (namely the new AWS Batch service, infrastructure-as-code, and containerized workflows) would greatly improve its utility. Thus, in 2019 HyP3 was re-designed to exist entirely within the AWS cloud, with application-specific processing workflows encapsulated within Docker containers. The re-design was completed in 2020, and on-demand SAR image processing services that utilized HyP3 were made available to the public for free via an API, Python SDK, and ASF's Data Search Vertex website.
+By 2019 it was clear that re-designing HyP3 to take advantage of evolving technologies (namely the new AWS Batch service, infrastructure-as-code, and containerized workflows) would greatly improve its utility. Thus, in 2019 HyP3 was re-designed to run entirely within the AWS cloud, with application-specific processing workflows encapsulated within Docker containers. The re-design was completed in 2020, and on-demand SAR image processing services that utilized HyP3 were made available to the public at no cost via an API, Python SDK, and ASF's Data Search Vertex website.
```
Comment (hjkristenson) [inline paper/paper.md:46]: The "for free" vs. "at no cost" is one we've struggled with a lot in our documentation. Maybe "made publicly available at no cost to users via an API"... might work?

Comment (jtherrmann) [reply paper/paper.md:46]: Yeah, the term "free" is often used in the context of "free software", meaning software provided under a libre license. Of course, that [also applies to HyP3](https://github.com/ASFHyP3/hyp3/blob/develop/LICENSE), but I assume that in this context we want to specifically highlight the no-cost aspect of HyP3. So I think "at no cost" is generally more explicit and a bit more readable than "free".

File: paper/paper.md (hunk 1 above)
Comment (jtherrmann) [reply paper/paper.md:44]: I didn't realize this was even a debate, I just assumed that the [Wikipedia article](https://en.wikipedia.org/wiki/On-premises_software) probably knew best.

**Review by jhkennedy — APPROVED** [review]
These all look good to me!

**Comment by forrestfwilliams** [comment]
@jtherrmann I didn't realize that the student HyP3 team wasn't affiliated with UAF. were you all going to a different school?

**Review by forrestfwilliams — COMMENTED** [review]
Merged!

**Comment by jtherrmann** [comment]
@forrestfwilliams Thanks! Yeah, the three of us were all at different universities at that point. I eventually transferred to UAF a few years later.

**Pull request merged**